platform/binance/model.go

```diff
@@ -266,6 +266,10 @@ func (e *Error) Error() string {
 
 // Determines Explorer transaction direction relatively to address
 func (tx *ExplorerTxs) Direction(address string) blockatlas.Direction {
+	if address == "" {
+		return ""
```

Comment:
can you return `blockatlas.Direction` type here?

Reply:
It is blockatls.Direction type. Golang do cast instead of me automatically. Because return type is blockatlas.Direction is string